src/rgw/rgw_rados.cc

```diff
@@ -4658,10 +4661,7 @@ int RGWRados::init_complete()
   }
   ldout(cct, 20) << __func__ << " bucket index max shards: " << bucket_index_max_shards << dendl;
 
-  binfo_cache = new RGWChainedCacheImpl<bucket_info_entry>;
-  binfo_cache->init(this);
-
-  bool need_tombstone_cache = !zone_data_notify_to_map.empty(); /* have zones syncing from us */
+  bool need_tombstone_cache = !svc.zone->get_zone_data_notify_to_map().empty(); /* have zones syncing from us */
```

Comment:
The original commit https://github.com/ceph/ceph/commit/78125a8dba6b59b012c642c6b756a2c4f34912b1 does not touch this line, so the backport should not touch it, either.